Adds link to privacy policy

p-galligan opened the pull request:
Adds link to RAC privacy policy. Fixes #100 . Keeping it extremely simple right now. Just adds it over the theme's footer text.

<img width="1440" alt="Screen Shot 2020-08-12 at 11 24 02 AM" src="https://user-images.githubusercontent.com/2585069/90034415-97e42700-dc8e-11ea-814b-e67f25d03519.png">

Commit:
add RAC Privacy Policy

Adds link to RAC privacy policy.

Commit:
update formatting

tweaks formatting slightly

Commit:
update bundler

updates bundler version

Commit:
add kramdown parser

adds kramdown parser gem

helrond commented on _includes/footer.html:
```diff
@@ -5,6 +5,9 @@
 {% if site.theme_settings.footer_text %}
 <footer>
 	<div class="site-footer">
+	<p class="text">
+		<a rel="RAC Privacy Policy" href="https://rockarch.org/about-us/privacy-policy/" style="text-decoration:underline">RAC Privacy Policy</a>
+	</p>
```
This seems overly complex. Can we do something like:
```suggestion
	<a href="https://rockarch.org/about-us/privacy-policy/">Privacy Policy</a>
```
If styling needs to happen it should be done via a class on the anchor tag, not as inline styles.

@HaSistrunk can correct me if I'm wrong but I believe that the `rel` attribute is not necessary if link text is present.

p-galligan replied:
I am definitely no CSS wiz, so this does look better to me. The only comment I'd have is the possible addition of underlining because the color contrast isn't super high on this site.

helrond replied:
You can do that by adding a CSS rule. The inline styles on the links in the footer are Bad Form and this would be a good opportunity to fix those.

Commit:
simplify privacy policy

simplifies link to privacy policy and adds underlining to all `<a>` tags in the footer.

helrond reviewed (approved):
That looks more like it!

HaSistrunk commented:
Yeah, nice!
